__tests__/app.test.js

```diff
@@ -126,15 +126,60 @@ describe("GET  /api/articles/:article_id/comments", () => {
 
   test("404: should return not found when not valid number passed", () => {
     return request(app)
-      .get("/api/articles/13/comments")
+      .get("/api/articles/13/comment")
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe("not found");
       });
   });
   test("400: should return bad request when not non number passed", () => {
     return request(app)
-      .get("/api/articles/nonsense/comments")
+      .get("/api/articles/nonsense/comment")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
```

Comment:
What would happen if the user tried to post a comment with an empty body? Implement this in your code

Reply:
i also implement  test for if only username is post